README.md

```diff
@@ -127,17 +145,28 @@ Keep the console open while testing. If you close it, the tunnel also closes and
 
 1. Expose a public HTTPS notification endpoint. It can run on a service such as Microsoft Azure, or you can create a proxy web server by [using ngrok](#ngrok) or a similar tool.
 
-1. Open the **GraphWebhooks-Core.sln** sample file in Visual Studio 2015. 
+1. Open the **GraphWebhooks-Core.sln** sample file in Visual Studio 2017. 
 
 1. In Solution Explorer, open the **appsettings.json** file in the root directory of the project.  
-   a. For the **AppId** key, replace *ENTER_YOUR_APP_ID* with the application ID of your registered Azure application.  
-   b. For the **AppSecret** key, replace *ENTER_YOUR_SECRET* with the key of your registered Azure application. Note that in production apps you should always use certificates as your application secrets, but for this sample we will use a simple shared secret password.  
-   c. For the **NotificationUrl** key, replace *ENTER_YOUR_URL* with the HTTPS URL. Keep the */notification/listen* portion. 
    
+   - For the **NotificationUrl** key, replace *ENTER_YOUR_URL* with the HTTPS URL. Keep the */notification/listen* portion. 
+
    If you're using ngrok, use the HTTPS URL that you copied. The value will look something like this:
 
    `"NotificationUrl": "https://2885f9c5.ngrok.io/notification/listen",`

```

Comment:
You might want to explain a bit more how/why you'd use it? (not clear to me)

Reply:
Updated as below:

`This is the url endpoint that will receive subscription validation callbacks and notification events from Graph, through the proxy server set up above (ngrok, for this sample).`